.github/workflows/cicd.yaml

```diff
@@ -0,0 +1,192 @@
+      env:
+        NGC_REPO: nvstaging/mellanox/network-operator
+      run: |
+        make chart-build chart-push
+    - if: github.ref_name == env.DEFAULT_BRANCH || github.ref_type == 'tag'
+      name: NGC logout
+      run: |
+        ngc config clear-cache
+        ngc config clear
+
+  ocp-bundle:
+    if: github.ref_type == 'tag'
+    needs:
+    - docker-build-push
+    runs-on: ubuntu-latest
+    env:
+      GH_TOKEN: ${{ secrets.GH_TOKEN_NVIDIA_CI_CD }}
+      DOWNSTREAM_REPO_OWNER: nvidia-ci-cd
+      UPSTREAM_REPO_OWNER: redhat-openshift-ecosystem
+    steps:
+    - uses: actions/checkout@v4
+      with:
+        token: ${{ secrets.GH_TOKEN_NVIDIA_CI_CD }}  # token must be explicitly set here for push to work in following step
+    - uses: docker/login-action@v3
+      with:
+        registry: ${{ env.REGISTRY }}
+        username: ${{ secrets.NVCR_USERNAME }}
+        password: ${{ secrets.NVCR_TOKEN }}
+    - name: Determine version and tag
+      run: |
+        git_tag=${{ github.ref_name }}
+        echo VERSION_WITH_PREFIX=$git_tag        >> $GITHUB_ENV
+        echo VERSION_WITHOUT_PREFIX=${git_tag:1} >> $GITHUB_ENV  # without the 'v' prefix
+    - name: Lookup image digest
+      run: |
+        network_operator_digest=$(skopeo inspect docker://$REGISTRY/$IMAGE_NAME:$VERSION_WITH_PREFIX | jq -r .Digest)
+        echo $network_operator_digest | wc -w | grep 1  # verifies value not empty
+        echo NETWORK_OPERATOR_DIGEST=$network_operator_digest >> $GITHUB_ENV
+    - name: Make bundle
+      env:
+        DEFAULT_CHANNEL: ${{ env.VERSION_WITH_PREFIX }}
+        CHANNELS: stable,${{ env.VERSION_WITH_PREFIX }}
+        TAG: ${{ env.REGISTRY }}/${{ env.IMAGE_NAME }}@${{ env.NETWORK_OPERATOR_DIGEST }}
+        BUNDLE_IMG: ${{ env.REGISTRY }}/${{ env.IMAGE_NAME }}-bundle:${{ env.VERSION_WITH_PREFIX }}
+        VERSION: ${{ env.VERSION_WITHOUT_PREFIX }}
+        NGC_CLI_API_KEY: ${{ secrets.NVCR_TOKEN }}
+      run: |
+        make bundle bundle-build bundle-push
+    - name: Create PR with bundle to Network Operator
+      env:
+        FEATURE_BRANCH: update-ocp-bundle-to-${{ env.VERSION_WITH_PREFIX }}
+      run: |
+        git config user.name  nvidia-ci-cd
+        git config user.email svc-cloud-orch-gh@nvidia.com
+
+        git checkout -b $FEATURE_BRANCH
+        git status
+        git add bundle
+        git commit -m "task: update bundle to $VERSION_WITH_PREFIX"
+
+        git push -u origin $FEATURE_BRANCH
+        gh pr create \
+          --head $FEATURE_BRANCH \
+          --base $DEFAULT_BRANCH \
+          --title "task: update bundle to $VERSION_WITH_PREFIX" \
+          --body "Created by the *${{ github.job }}* job in [${{ github.repository }} OCP bundle CI](${{ github.server_url }}/${{ github.repository }}/actions/runs/${{ github.run_id }})."
+        # gh pr merge --admin --merge $FEATURE_BRANCH  # requires repository settings to "allow auto-merge"  # TODO: only auto-merge once proper tests are implemented
```

Comment:
Remove this line.
Even when we will have tests and decide to auto-merge, it will not be merged here, but in a cron job checking if tests passed before merging.